### DISCUSSION TITLE
Civet test failing on "Build framework" step

### DISCUSSION THREAD

**Discussion by smpark7** [idaholab/moose#25283] [Q&A Tools] [answered]
I'm working on a [PR](https://github.com/arfc/moltres/pull/231) for [Moltres](https://github.com/arfc/moltres) which introduces new tests and documentation but doesn't touch any source code.

The [Civet CI](https://civet.inl.gov/pr/18529/) keeps failing on the "Build framework" step with the following text:
```
//: Running in versioned apptainer container moose-dev
//: Container  not found
//: Container  not found
Failed to find a container to run in
```
The CI tests for other PRs run fine without any issues. Can I safely ignore this failed test and merge my PR?

**Comment by GiudGiud** [comment]
@loganharbour @milljm

**Reply by GiudGiud** [reply]
I would say no, you need to run the test suite. The new tests could be failing in a tested configuration and the new docs could have defects that would only be picked up by a documentation test build.

**Reply by loganharbour** [reply, marked as answer]
The moose in moltres is so old that it doesn't support our CI anymore

**Reply by lindsayad** [reply]
So @smpark7 it sounds like you should update the MOOSE submodule hash as a first step

**Reply by smpark7** [reply]
Ah I see. Thanks for checking this!
